riva/clients/tts/riva_tts_client.cc

```diff
@@ -176,8 +175,7 @@ main(int argc, char** argv)
         std::chrono::duration<double> elapsed_first_audio = t_first_audio - start;
         std::cerr << "Time to first chunk: " << elapsed_first_audio.count() << " s" << std::endl;
       }
-      std::cerr << "Got chunk: " << chunk.audio().size() << " bytes" << std::endl;
-
+      std::cout << "Got chunk: " << chunk.audio().size() << " bytes" << std::endl;
```

Comment:
Why aren't we using glog here?

Reply:
Idk, it's everywhere and I just follow the pattern